src/support.ts

```diff
@@ -4,9 +4,11 @@ import { realSwipe } from "./commands/realSwipe";
 import { realPress } from "./commands/realPress";
 import { realType } from "./commands/realType";
 import { realTouch } from "./commands/realTouch";
-import { realMouseDown } from "./commands/mouseDown";
-import { realMouseUp } from "./commands/mouseUp";
+import { realMouseDown } from "./commands/realMouseDown";
+import { realMouseUp } from "./commands/realMouseUp";
+import { InternalState } from './_internalState'
 
+Cypress.on('window:before:load', InternalState.clear)
```

Comment:
It seems like a reasonable hook to clear the state. I think that this should work (although it's weird):
```
it('foo', () => {
  cy.visit('https://...')
  cy.get('.btn').mouseDown()
})
it('bar', () => {
  // no visit or anything, piggybacking on the state from the previous test block
  // performing anything here should assume that the mouse button is still down
})
```